Backend/Backend/Models/User.cs

```diff
@@ -1,17 +1,29 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using Microsoft.EntityFrameworkCore;
 
-namespace backend.Models;
+namespace Backend.Models;
 
+/// <summary>
+/// User model.
+/// </summary>
 public class User : BaseEntity
 {
+    /// <summary>
+    /// User Unique Identifier.
+    /// </summary>
     [Key]
     public Guid Id { get; set; }
     
+    /// <summary>
+    /// User email.
+    /// </summary>
     [Required]
     [EmailAddress]
     public string? Email { get; set; }
 
+    /// <summary>
+    /// User password.
+    /// </summary>
     public string? Password { get; set; }
```

Comment:
Should probably change this not be a nullable, or at least make it required. Same with Email I think

Reply:
Added the password as required. Also added a default constructor to allow for the string properties to be set as not nullable.